packages/web-components-stencil/src/components/alert/alert.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component, Element, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'lux-alert',
+  styleUrl: 'alert.scss',
+  shadow: true,
+})
+export class Alert {
+  @Prop() type?: string;
+  @Prop() hideIcon? = false;
+  @Element() hostElement!: HTMLElement;
+
+  // private utrechtAlertElement?: HTMLUtrechtAlertElement;
+
+  render() {
+    const { type, hideIcon } = this;
+
+    return (
+      <utrecht-alert type={type} part="utrecht-alert">
+        {!hideIcon ? (
```

Comment:
Volgens mij is deze vlag niet nodig. Als een slot geen element krijgt toegekend is het gewoon een leeg element zonder styling

Reply:
Oh wacht. Ik zie het al. Je laat het default icoon zien o.b.v. het `type` attribuut, maar biedt de mogelijkheid een eigen icoon weer te geven. Is dat toegestaan door UX?

Reply:
UX heeft standaard een icoon in de designs staan. Bij Utrecht niet en moet je handmatig een icoon toevoegen.
@AlineNap wat is jou idee mbt tot het aanbieden dat er geen icoon wordt getoond. IMHO zou het ook zonder icoon moeten kunnen.

Reply:
We hebben het hier offline over gehad. Daaruit kwam dat er altijd een icon moet staan, ook al heeft Gemeente Utrecht dat niet. D.w.z. dat het slot weg kan en we een keuze voor `type` property moeten afdwingen.